src/Microsoft.Identity.Web/CertificateManagement/DefaultCertificateLoader.cs

```diff
@@ -94,7 +94,7 @@ private static X509Certificate2 LoadFromKeyVault(string keyVaultUrl, string cert
             }
 
             // Parse the secret ID and version to retrieve the private key.
-            string[] segments = certificate.SecretId.AbsolutePath.Split('/', StringSplitOptions.RemoveEmptyEntries);
+            string[] segments = certificate.SecretId.AbsolutePath.Split(new[] { '/' }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Why don't you leave the string.Split as is ?

Reply:
Because the particular override that was used is available in netcoreapp3.1 and net5.0, but not in net461/net472